#166: Only normalize OffsetDateTimes when they are normalizable

morth opened the pull request:
Possible fix for #166. This commit adds a check before the OffsetDateTime adjustment of ADJUST_DATES_TO_CONTEXT_TIME_ZONE that ensures that the OffsetDateTime is normalizeable given a specific time zone offset. When it is not, normalization is skipped and the un-adjusted OffsetDateTime is returned.

cowtowncoder commented:
+1 for suggestion on static member. One other suggestion: if easy, rebasing against 2.11 (if considered safe change) or 2.12 (if there is risk) would be nice. That can then be merged forward 2.11 -> 2.12 -> master (3.0). Not a big deal as cherry-picking back from `master` is probably easy as well.

morth commented:
@kupci @cowtowncoder I implemented your suggestions and fixed this in 2.11 in #173.

@kupci 'Ex' was for "excluded", since it was the first value outside of the boundary.
